### DISCUSSION TITLE
generator usually makes duplicated data

### DISCUSSION THREAD

**Discussion by rumjie** [Diyago/Tabular-data-generation#51] [Q&A]
I have dataframe 1000 rows and 27 cols and most of columns are categorical. By using tab gan and 4~5 rows of df are exactly same. I don't think it will be helpful to models - is this relative with categorical data or lack of data size?

**Comment by adesh14** [comment]
hey, 
 I am encountering an error like [LightGBM] [Warning] No further splits with positive gain, best gain: -inf 
my dataset contains 360 rows and 19999 col can you please guide what could be the problem
